boxes/identity-server-analytics/provisioner/provisioner.sh

```diff
@@ -59,12 +65,20 @@ if test -d ${JAVA_HOME}; then
 fi
 echo "Successfully set up Java"
 
-# unpack the WSO2 product pack to the working directory
-echo "Setting up the ${WSO2_SERVER}-${WSO2_SERVER_VERSION} server..."
+# set up WUM
+echo "Setting up WUM..."
+if test ! -d ${WUM_HOME}; then mkdir ${WUM_HOME}; fi
+if test -d ${WUM_HOME}; then
+  tar -xzf ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} -C ${WUM_HOME} --strip-components=1
```

Comment:
@VimukthiPerera why do we need `z` argument for extracting the WSO2 WUM distribution in this case? I believe `tar -xf ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} -C ${WUM_HOME} --strip-components=1` would suffice.

Reply:
@chirangaalwis yes. After version  1.15 "z" argument is not needed. But in the WUM installation guide it was mentioned as above https://wso2.com/wum/download . I'll change this to "-xf"